integration/tests/a1migration.sh

```diff
@@ -46,7 +46,7 @@ do_test_deploy() {
     chef_server_migration_smoke_tests
     ## skipping status test because of the missing file in automate - /etc/opscode/chef-server-running.json 
```

Comment:
It would be useful to add the status to pedant config so we are not depending on the /etc/opscode/chef-server-running.json  (Not a blocker for this PR.